examples/jsm/loaders/EXRLoader.js

```diff
@@ -1991,6 +2041,194 @@ class EXRLoader extends DataTextureLoader {
 
 		}
 
+		function floorLog2( x ) {
```

Comment:
Out of curiosity: This could be implemented as `Math.floor( Math.log2( x ) );` as well. Is your version more performant?

Reply:
I have asked myself the same question, both `floorLog2` and `ceilLog2` were taken directly from OpenEXR's [impl](https://github.com/AcademySoftwareFoundation/openexr/blob/9d3e061a0b4090f268eee50e84fcccf5cbcad9ed/src/lib/OpenEXR/ImfTiledMisc.cpp#L143-L180). I even considered using `31 - Math.clz32( x )` which is also the same thing. 

Given that this piece of code is only running a couple of times during header parsing and isn't exactly performance critical, I decided to just err on the side of safety and copy their impl. But you are right, it isn't very performant [jsperf](https://jsperf.app/mobapu)

I have taken a few notes for places where I found possible optimizations, and I was planning on testing those after I'm done with basic features implementation. This case one of them, but I can patch this now, if you prefer.